### DISCUSSION TITLE
Subscription not connecting

### DISCUSSION THREAD

**Discussion by Topener** [supabase/supabase#14849] [Questions]
Hi all,

I have a new instance set up (since a week) and I'm trying to get subscriptions to work in my JavaScript/NodeJS project. However, I'm running into the issue that the subscription is not working/connected. Not sure how to get it working.

I'm on a local environment, and regular queries seem to work just fine. Take this code (simplified) it works great:

```js
    const { data, error } = await supabase.from("user_tokens").select("*");
    data.forEach((userToken) => {
        userTokens[userToken.user] = userToken.tokens;
    });
```

After this code I'm trying to subscribe to all events in the same table with the example code provided on the docs, however, I never get any messages when I change a cell in the table. I also see no indication any connection was ever established on the dashboard. I do however have replication enabled for this particular table.

```js
    const channel = supabase
        .channel("table-db-changes")
        .on(
            "postgres_changes",
            {
                event: "*",
                schema: "public",
                table: "user_tokens",
            },
            (payload) => console.log(payload)
        )
        .subscribe();
```

Thoughts how to proceed? Thanks!

**Comment by GaryAustin1** [comment]
You can add `.subscribe(status=>console.log(status)) ` to possibly get information.  You should also verify your table has realtime enabled.

You can also use https://realtime.supabase.com/inspector/new to check your instance realtime connection separately from your code.

**Comment by yirenlu92** [comment]
I'm having the same issue - using the sample code from the documentation, it was working fine a week ago.

I tried adding `.subscribe(status=>console.log(status))` and it showed that the channel had been SUBSCRIBED to.

I then tried the inspector to check the realtime connection and this is what it showed:

![Screen Shot 2023-06-06 at 6 04 05 PM](https://github.com/supabase/supabase/assets/13631199/bb0700c9-fc8b-4671-8eb1-634a7ad5fe6e)

So it looks like database changes are not subscribed to?

Is this something I have to configure separately?

**Comment by Topener** [comment]
The realtime inspector is connecting, but not seeing any events when I change a cell in the dashboard. My code is giving me`CHANNEL_ERROR` though I have yet to find a good cause for this. 

I have replication enabled for this table, and all switches are on at the `/database/replication` page.
